BLSQ/iaso
# IA-2897 - Registry v3

By default the user lands on the Org unit shown on the left (on the mockup, District Yoko) but upon click on the map, the user can see the information relating to the children org unit on the right panel. 

On the right panel, add a breadcrumb from the landing Parent to the selected children which allows to also go back to the Parent as needed


By default one of the Reference forms opens (so there is always data showing)
Related JIRA tickets : IA-2897

## Self proofreading checklist




## Changes

- refactor map and submissions / org unit components into smaller components
- use tree picker with an icon in breadcrumb
- use skeleton component to fill up page if a default org unit hasn't been selected
- display children selected info to the right, allowing to display forms submissions
- remove map popup
- align map and submissions on the same height
- make sure selected children is deep linked

## How to test

You need a good org unit type config with levels and sub types.
Make sure the org unit you are going to test have geolocation and a submission (reference or not) linked to it
go to registry page and select the root org unit.

Play with the page, drill down , drill up using breadcrumb or map.
Check that submissions are correct on the right side

## Print screen / video

https://github.com/BLSQ/iaso/assets/12494624/7e31c9e1-5a99-4356-a54e-b0787235b79e
